kafka/make_credentials.sh

```diff
@@ -0,0 +1,33 @@
+source .env
+
+function gen_kafka_creds {
+    cat << EOF
+KafkaServer {
+   org.apache.kafka.common.security.scram.ScramLoginModule required;
```

Comment:
Are these semicolons correct? If so, that's just plain weird 😕

Reply:
I agree, but they are. They separate sections (Scram from Plain) in this case.